cockatrice/src/gamesmodel.h

```diff
@@ -9,6 +9,16 @@
 #include <QList>
 #include <QSet>
 #include <QSortFilterProxyModel>
+#include <QStringList>
+#include <QTime>
+
+const bool DEFAULT_UNAVAILABLE_GAMES_VISIBLE = false;
+const bool DEFAULT_SHOW_PASSWORD_PROTECTED_GAMES = true;
+const bool DEFAULT_SHOW_BUDDIES_ONLY_GAMES = true;
+const bool DEFAULT_HIDE_IGNORED_USER_GAMES = false;
+const int DEFAULT_MAX_PLAYERS_MIN = 1;
+const int DEFAULT_MAX_PLAYERS_MAX = 99;
+constexpr QTime DEFAULT_MAX_GAME_AGE = QTime();
```

Comment:
we might consider moving these to the implementation file instead, what do you think?

Reply:
Good idea. Done!